content/it/docs/concepts/containers/images.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Immagini
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+L'immagine di un container rappresenta dati binari che incapsulano un'applicazione e 
+tutte le sue dipendenze software. Le immagini sono costituite da pacchetti software 
+eseguibili che possono essere avviati in modalità standalone e su cui si possono fare 
+ipotesi ben precise circa l'ambiente in cui vengono eseguiti.
+
+Tipicamente viene creata un'immagine di un'applicazione ed effettuato il _push_ 
+su un registro prima di poterne fare riferimento esplicito in un 
```

Comment:
```suggestion
su un registry (un repository pubblico di immagini) prima di poterne fare riferimento esplicito in un 
```
FYI, abbiamo concordato che è possibile usare termini inglesi quando li usiamo comunemente nel gergo che gli addetti IT usano in Italia. In questo caso, suggerisco di usare registry, seguito dalla sue definizione tra parentesi, invece di registro che può risultare troppo generico.